graspy/classify/base.py

```diff
@@ -0,0 +1,66 @@
+# base.py
```

Comment:
make header compliant with the new headers (can just copy and paste from a file in master)